src/styles/blocks/_catalog.scss

```diff
@@ -0,0 +1,37 @@
+.catalog {
+  display: grid;
+  justify-content: center;
+  gap: 48px;
+  margin: 0 auto;
+
+  @media(min-width: 1024px) {
+    grid-template-columns: repeat(4, auto);
```

Comment:
Nothing wrong, just a suggestion, this can be put directly under .catalog, since it will only get displayed if none of the other resolutions are true

Reply:
You mean 1024 resolution part?

Reply:
I mean grid-template-columns: repeat(4, auto)

Reply:
Got it.